README.md

```diff
@@ -391,6 +391,29 @@ network_groups:
   - name: "reporting-apps"
     networks: ["10.10.10.0/24"]
 
+# Optional list of GET params to send with each request
```

Comment:
substitute this with the following text:

    Optional lists of query params to send with each proxied request to ClickHouse.
    These lists may be used for overriding ClickHouse settings on a per-user basis.